nixpkgs_review/report.py

```diff
@@ -350,6 +381,19 @@ def markdown(self, pr: int | None) -> str:
             )
             msg += html_pkgs_section(":white_check_mark:", report.built, "built")
 
+        if self.show_logs:
+            for system, report in self.system_reports.items():
+                if not report.failed:
+                    continue
+                full_msg = msg
+                full_msg += "\n---\n"
+                full_msg += f"### Error logs: `{system}`\n"
+                full_msg += html_logs_section(get_log_dir(root), report.failed, system)
+                # if the final message won't fit a single github comment, stop
+                if len(full_msg) > MAX_GITHUB_COMMENT_LENGTH:
```

Comment:
Could we leave a message that we not included all logs due to length restrictions?

```suggestion
                if len(full_msg) + len(logs_truncated_msg) > MAX_GITHUB_COMMENT_LENGTH:
```

Reply:
It will complicate bookkeeping somewhat since we'd have to track if the warning itself would have space in the report if it were to be appended.  But I will see what I can do.

Reply:
It's just because I can already foresee bug reports where people complain about missing logs.

Reply:
OK fair. (I was thinking about the edge case where the final log WOULD fit in the report - if only we'd not pre-accommodate for the warning message. But I guess I'm overthinking.)